src/main/java/edu/cornell/kfs/fp/batch/service/impl/AccountingXmlDocumentDownloadAttachmentServiceImpl.java

```diff
@@ -70,6 +70,25 @@ public Attachment createAttachmentFromBackupLink(Document document, AccountingXm
         }      
     }
 
+    protected String findMimeType(String uploadFileName) {
+        String mimeType = URLConnection.guessContentTypeFromName(uploadFileName);
+        if (StringUtils.isEmpty(mimeType)) {
+            LOG.error("findMimeType, could not determine mime type from file name using URLConnection object. the file name is " + uploadFileName);
+            String[] splitFileName = StringUtils.split(uploadFileName, ".");
+            if (splitFileName != null) {
+                int lastArrayElement = splitFileName.length - 1;
```

Comment:
This variable should be renamed, to make it clear that it holds the element index and not the element itself.

Reply:
can do